kubeflow/core/setup_iap.sh

```diff
@@ -68,9 +71,29 @@ echo BACKEND_ID=${BACKEND_ID}
 NODE_PORT=$(kubectl --namespace=${NAMESPACE} get svc ${SERVICE} -o jsonpath='{.spec.ports[0].nodePort}')
 BACKEND_SERVICE=$(gcloud --project=${PROJECT} compute backend-services list --filter=name~k8s-be-${NODE_PORT}- --uri)
 # Enable IAP on the backend service:
-gcloud --project=${PROJECT} compute backend-services update ${BACKEND_SERVICE} \
-    --global \
-    --iap=enabled,oauth2-client-id=${CLIENT_ID},oauth2-client-secret=${CLIENT_SECRET}
+iap_patch='.iap = {
+  "enabled": true,
+  "oauth2ClientInfo": {
+    "developerEmailAddress": "iap@kubeflow.org",
```

Comment:
Can we use an environment variable to pass in the user's email address and not a kubeflow.org email?

Reply:
Done